docs/CodeTransformation.md

```diff
@@ -30,12 +30,18 @@ interface Transformer<OptionType = unknown> {
    * If V8 coverage is _not_ active, and this is `false`. Jest will instrument the code returned by this transformer using Babel.
    */
   canInstrument?: boolean;
-  createTransformer?: (options?: OptionType) => Transformer;
+  createTransformer?: (options: OptionType) => Transformer;
 
   getCacheKey?: (
     sourceText: string,
     sourcePath: string,
     options: TransformOptions,
+    /**
+     * If a transformer does module resolution and reads files, it should populate `cacheFS` so that
+     * Jest avoids reading the same files again, improving performance. `cacheFS` stores entries of
+     * <file path, file contents>
+     */
+    cacheFS: Map<string, string>,
```

Comment:
It should be added to `TransformOptions` (which will then make it be included in `process`).

I wanted to get away from more args in #10834

Reply:
I made it as a mandatory property of `TransformOption`, hope it's ok since it requires `jest-repl` to be adjusted too.

Reply:
That's fine 👍